assets/js/common/PatchList/PatchList.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  EOS_SHIELD_OUTLINED,
+  EOS_CRITICAL_BUG_OUTLINED,
+  EOS_ADD_BOX_OUTLINED,
+} from 'eos-icons-react';
+import { format as formatDate } from 'date-fns';
+import classNames from 'classnames';
+
+import Table from '@common/Table';
+import { computedIconCssClass } from '@lib/icon';
+
+function iconFromAdvisoryType(
+  advisoryType,
+  centered = false,
+  hoverOpacity = true,
+  size = 'l'
+) {
```

Comment:
Try to use arrow functions!

```suggestion
const iconFromAdvisoryType = (
  advisoryType,
  centered = false,
  hoverOpacity = true,
  size = 'l'
) => {
```

Reply:
Just wondering, what's the reason for this?

Reply:
Should the function be uppercased to, since it returns a compoenent instance?

Reply:
We usually prefer arrow functions since they have lower memory footprint, even tho they get transpiled :smile: No need for an uppercase!